pkg/pgmodel/testdata/expected/info_view-postgres.out

```diff
@@ -37,12 +37,12 @@ SELECT id , metric_name , table_name, retention_period, chunk_interval > interva
 (2 rows)
```

Comment:
We should add queries to prom_info.metric_stats and prom_info.system_stats and the label_cardinality function to these tests